tests/common.go

```diff
@@ -7786,6 +7790,8 @@ func CreateMultiVolumesAndAttach(wg *sync.WaitGroup, count int, nodeName string)
 		createdVolIDs[volId] = volPath
 		log.Infof("Volume %s attached to path %s", volId, volPath)
 		count--
+		log.InfoD("Printing the volume inspect for the volume")
```

Comment:
How about log.Debugf and include volume details as well?

Reply:
changed it, thanks for notifying